### ISSUE TITLE
Running `eval` out of order results in a perma-hang

### ISSUE THREAD

**Issue by jnoller** [getcarina/carina#98]
If you have the muscle memory / history you may skip the `carina credentials cluster` call and jump to the eval which results in a forever hang:

```
[ jesse@pug ] ~$> carina create --wait cluster
ClusterName         Flavor              Segments            AutoScale           Status
cluster         container1-4G       1                   false               active
[ jesse@pug ] ~$> eval $(carina env cluster)
```

At this point it hangs and you have to control-c

**Comment by carolynvs** [comment]
I've hit this a few times too but never realized it was because I scorn the `carina credentials` command. :smile:

**Comment by rgbkrk** [comment]
Env should be loading the credentials when not loaded which will hang to download.

**Comment by jnoller** [comment]
Not forever which is what it did 😨

**Comment by rgbkrk** [comment]
Definitely not a window resize event.

**Comment by carolynvs** [comment]
I believe this has been fixed in the v2.0. Please re-open if it comes up again.
